.travis.yml

```
@@ -4,24 +4,33 @@ language: haskell
 git:
   depth: 5
 
-cabal: "2.4"
+cabal: "3.0"
 
 cache:
   directories:
   - "$HOME/.cabal/store"
-  
+
+addons:
+  apt:
+    sources:
+    - sourceline: 'ppa:hvr/ghcjs'
+    update: true
+    packages:
+    - ghcjs-8.4
 
 matrix:
   include:
-  - ghc: 8.6.5
-  
+  - ghcjs: 8.4.4
```

For some reason it's installed GHC-7.10 (probably the default one). And this could result into some incompatibility between `base` versions.
```suggestion
  - ghc: 8.4.4
```